components/Header/Header.tsx

```diff
@@ -0,0 +1,43 @@
+import Image from 'next/image';
+import * as S from './Header.style';
+import IMAGES from '@/public';
+import Link from 'next/link';
+import axios from '@/lib/axios';
+import { useEffect, useState } from 'react';
+
+interface IUser {
+  email: string;
+  profileImageSource: string;
+}
+
+const Header = () => {
+  const [user, setUser] = useState({
```

Comment:
혹시 여기서 타입 단언을 해주신 이유가 있을까요?
타입 단언 없이 이렇게 써주셔도 무방합니다 :)

```suggestion
  const [user, setUser] = useState<IUser>({
    email: '',
    profileImageSource: '',
  });
```